core/src/main/scala/org/apache/spark/internal/io/SparkHadoopMapReduceWriter.scala

```diff
@@ -119,7 +119,7 @@ object SparkHadoopMapReduceWriter extends Logging {
     }
   }
 
-  /** Write a RDD partition out in a single Spark task. */
+  /** Write an RDD partition out in a single Spark task. */
```

Comment:
Are you saying the one below

```
./R/pkg/inst/tests/testthat/test_sparkSQL.R:test_that("jsonRDD() on a RDD with json string", {
```

?

If so, I intentionally excluded this as this is the change for documentation and that was not the part of documentation. Maybe, I was trying to be too cautious. Will include them in a minor PR i am going to submit.